dashboard/app/views/shared/_sharing_meta_tags.html.haml

```diff
@@ -0,0 +1,39 @@
+-# only works for pages that have a @level
+- if @level && @level.game.supports_sharing?
+  - content_for :head do
+    -# facebook boilerplate
+    = tag('meta', :property => 'fb:app_id', :content => '500177453358606')
+    = tag('meta', :property => 'og:site_name', :content => 'Code.org')
+    = tag('meta', :property => 'og:type', :content => 'article')
+    = tag('meta', :property => 'article:publisher', :content => 'https://www.facebook.com/Code.org')
+
+    -# twitter boilerplate
+    = tag('meta', :name => 'twitter:site', :content => '@codeorg')
+    = tag('meta', :name => 'twitter:card', :content => 'photo')
+
+    - if @level.game.app == Game::ARTIST
+      - meta_image_width = 500
+      - meta_image_height = 261
+    - else
+      - meta_image_width = 400
+      - meta_image_height = 400
+
+    -# several meta tags are the same for facebook/og and twitter
+    - {og: 'property', twitter: 'name'}.each do |namespace, attr|
```

Comment:
this code was cutpasted from level_sources/show.html.haml except this refactor here to dedupe the tags that are common between twitter and facebook